docs/pages/database-access/guides/redshift-serverless.mdx

```diff
@@ -0,0 +1,400 @@
+description: How to configure Teleport Database Access with AWS Redshift Serverless.
+---
+
+This guide will help you to:
+
+- Set up Teleport to access your AWS Redshift Serverless instances.
+- Connect to your databases through Teleport.
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+![Teleport Database Access Redshift Self-Hosted](../../../img/database-access/guides/redshift_selfhosted_serverless.png)
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+![Teleport Database Access Redshift Cloud](../../../img/database-access/guides/redshift_cloud_serverless.png)
+</ScopedBlock>
+
+## Prerequisites
+
+- A pre-existing Teleport `(=teleport.version=)` cluster. If you don't already
+  have Teleport running, see [How to Choose a Teleport Edition](../../choose-an-edition/introduction.mdx)
+  to learn more about your options, and how to set up Teleport.
+- AWS account with a Redshift Serverless configuration and permissions to create
+  and attach IAM policies.
+- Command-line client `psql` installed and added to your system's `PATH` environment variable.
+- A host where you will run the Teleport Database Service. This guide assumes an
+  EC2 instance, and provides a corresponding example of access control.
+- (!docs/pages/includes/user-client-prereqs.mdx!)
+
+## Step 1/5. Create an IAM policy for Teleport
+
+<Admonition type="note" title="Example Access Control">
+This guide provides an example configuration of IAM access roles as a model,
+and uses an EC2 instance to serve the Teleport Database Access service.
+The level of access provided may not suit your needs, or may not fit your
+organization's access conventions. You should adjust the AWS IAM permissions to
+fit your needs.
+</Admonition>
+
+Teleport needs AWS IAM permissions to be able to:
+
+- Discover and register Redshift databases.
+- Manage IAM user or IAM role policies.
+
+If you don't already have one, create a role to apply to the EC2 instance that
+the Teleport Database Service will run on. This guide uses the example name
+`teleport-redshift-node`.
+
+Under **Trusted entity type** choose "AWS service", and under **Use case** choose "EC2".
+
+Create and attach a policy to the `teleport-redshift-node` role to allow Teleport
+to bootstrap access to AWS Redshift Serverless:
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Action": [
+                "iam:AttachRolePolicy",
+                "iam:PutRolePermissionsBoundary"
+            ],
+            "Resource": [
+                "arn:aws:iam::(=aws.aws_access_key=):role/teleport-redshift-node"
+            ],
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "iam:GetPolicy",
+                "iam:CreatePolicy",
+                "iam:TagPolicy",
+                "iam:ListPolicyVersions",
+                "iam:CreatePolicyVersion"
+            ],
+            "Resource": [
+                "*"
+            ],
```

Comment:
If we want minimal permission:
```suggestion
            "Resource": [
                "arn:aws:iam::(account-id):policy/DatabaseAccess",
                "arn:aws:iam::(account-id):policy/DatabaseAccessBoundary"
            ],
```

Note that the poilcy name can be changed by `--policy-name` flag from the bootstrap command. Say `teleport db configure /opt/teleport/teleport_db.yaml --policy-name STeveDatabaseAccess`, then the policy name in the IAM policy would be
```
                "arn:aws:iam::(account-id):policy/STeveDatabaseAccess",
                "arn:aws:iam::(account-id):policy/STeveDatabaseAccessBoundary"
```

Reply:
Rephrasing to make sure I follow: You're saying:

- We can reduce the scope of these permissions by defining the policies this role can create. The policy names in your example match those created by our bootstrap command.
- In order for this to work, we must also update the bootstrap command we use later in the guide to specify this policy name.

Is that correct?

Reply:
> We can reduce the scope of these permissions by defining the policies this role can create. The policy names in your example match those created by our bootstrap command.

Yes

> In order for this to work, we must also update the bootstrap command we use later in the guide to specify this policy name.

The `--policy-name` is an optional flag, if not specified, the value is `DatabaseAccess`. So we don't have to mention this, or maybe in a tip. Up to you what you think it's more appropriate.

Reply:
Updated and noted in a3d1684df72a0b5a16b3d08ef993f286847c1c37 - I elected not to discuss the `--policy-name` flag in this page.